Revision: 1731
          http://sourceforge.net/p/rdesktop/code/1731
Author:   hean01
Date:     2013-08-14 13:23:01 +0000 (Wed, 14 Aug 2013)
Log Message:
-----------
Bitmap cache precaching is broken, tries to create bitmaps without window
which results in a sigsegv.

Workaround for bug #367

Modified Paths:
--------------
    rdesktop/trunk/rdesktop.c

Modified: rdesktop/trunk/rdesktop.c
===================================================================
--- rdesktop/trunk/rdesktop.c	2013-08-14 12:56:52 UTC (rev 1730)
+++ rdesktop/trunk/rdesktop.c	2013-08-14 13:23:01 UTC (rev 1731)
@@ -87,7 +87,7 @@
 RD_BOOL g_sendmotion = True;
 RD_BOOL g_bitmap_cache = True;
 RD_BOOL g_bitmap_cache_persist_enable = False;
-RD_BOOL g_bitmap_cache_precache = True;
+RD_BOOL g_bitmap_cache_precache = False;
 RD_BOOL g_use_ctrl = True;
 RD_BOOL g_encryption = True;
 RD_BOOL g_encryption_initial = True;
